cmd/version.go

```diff
@@ -45,12 +45,18 @@ var versionCmd = &cobra.Command{
 		// Lets fetch the info about the server
 		serverInfo, err := util.GetOcClient().GetServerVersion()
 		util.CheckError(err, "")
+
+		// make sure we only include openshift info if we actually have it
+		openshiftStr := ""
+		if len(serverInfo.OpenShiftVersion) > 0 {
```

Comment:
Do we need to check for `serverInfo.OpenShiftVersion != nil`?

Reply:
`serverInfo.OpenShiftVersion` is an empty string by default, so we just check that :)

Reply:
It only get's set when the actual Openshift version was properly determined from the server

Reply:
ok :+1: